test-case-reporting/static/test-run-list.html

```diff
@@ -0,0 +1,17 @@
+<html ⚡>
+<head>
+  <title>Test runs</title>
+</head>
+<body>
+  <ul>
+    {{#testRuns}}
+    <li>Name is "{{testCase.name}}", job num is "{{job.jobNumber}}", status is {{status}}</li>
```

Comment:
Let's meet to discuss design of the views (which links & kinds of pages we will have).

Reply:
Let's start _extremely_ simple, just throw text on the page. Point of this PR is to get the infrastructure in place to render these two views, doesn't matter what they look like yet. That way we save the design and UX discussions for PR separate from the technicalities of Mustache.

Reply:
Does that look simple enough? Or, do you want me to remove the links

Reply:
LGTM